cueparser.py

```diff
@@ -253,11 +236,20 @@ def main():
         elif (args.number):
             num = int(args.number)
             print(cuesheet.getTrackByNumber(num))
+        elif (args.all):
+            print_all_tracks(cuesheet)
         else:
             print(cuesheet.output())
     except ValueError:
         print("Cannot parse int")
         exit()
 
+
+def print_all_tracks(cuesheet, i=0):
+    print("{0} tracks".format(len(cuesheet.tracks)))
```

Comment:
Have you forgot remove this line or you want print tracks count? If you want, may be it should be configurable.